pkg/kubernetes/kubernetes.go

```diff
@@ -43,14 +43,15 @@ func Init() error {
 
 	err = utils.RunCmdAndWait("kubectl", "apply", "-f", daprManifestPath)
 	if err != nil {
-		if s != nil {
-			s.Stop()
+		err = ensureConfig(client)
```

Comment:
if err happens not from configuration, what happens ?

Reply:
Since this is just applying Kubernetes resources, there's no other condition that can make it error out besides this.

Reply:
So, there's no way to know what caused this.

Reply:
I think we need to file the issue to figure it out the root cause... but for now, I agree that we can take this to release as a hotfix.

Reply:
root cause of what? we know why it's failing..

Reply:
The error messages we saw were very clear: It was unable to install the configuration object because the CRD resource wasn't registered yet

Reply:
And CRDs are the only dependencies in the installation manifest (and in general..)

Reply:
Ah ok I was confused. 

I think we need to check if the error is failing from configuration or somewhere else.

Reply:
There's no way we can check that..

Reply:
@yaron2 we can do with his pr - https://github.com/dapr/cli/commit/8699ed9a8ab688e205657c2dc7c768dc2b627e86 :) his change basically grab all console out :)

Reply:
Its shaky.. basing on output logs..